kollect-extensions/src/main/kotlin/kollect/extensions/Kollect.kt

```diff
@@ -0,0 +1,33 @@
+package kollect.extensions
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.core.Tuple2
+import arrow.extension
+import arrow.typeclasses.Monad
+import kollect.Kollect
+import kollect.KollectPartialOf
+import kollect.KollectResult
+import kollect.fix
+
+// Kollect ops
+@extension
+interface KollectMonad<F, I> : Monad<KollectPartialOf<F>> {
```

Comment:
This should be split up in seperate typeclasses.

In Scala having an instance for `Monad` is enough because the implicit system can resolve and provide instances for `Functor`, `Applicative`, etc which is transparent but in the Kotlin/Arrow API we have `Applicative<F>` etc all over as explicit params. I think it makes the API more user-friendly if you can call `Kollect.applicative()` etc depending on what you actually need.